```
/home/travis/catkin_ws/src/image_pipeline/image_view/src/nodelets/image_nodelet.cpp:165:5: error: ‘CvtColorForDisplayOptions’ is not a member of ‘cv_bridge’

     cv_bridge::CvtColorForDisplayOptions options;

     ^

/home/travis/catkin_ws/src/image_pipeline/image_view/src/nodelets/image_nodelet.cpp:165:42: error: expected ‘;’ before ‘options’

     cv_bridge::CvtColorForDisplayOptions options;

                                          ^

/home/travis/catkin_ws/src/image_pipeline/image_view/src/nodelets/image_nodelet.cpp:166:5: error: ‘options’ was not declared in this scope

     options.do_dynamic_scaling = do_dynamic_scaling;

```

--- a/image_view/cfg/ImageView.cfg
+++ b/image_view/cfg/ImageView.cfg
@@ -5,6 +5,23 @@ from dynamic_reconfigure.parameter_generator_catkin import *
 
 gen = ParameterGenerator()
 
+edit_method_colormap = gen.enum([
+    gen.const("NO_COLORMAP", int_t, -1, "NO_COLORMAP"),
+    gen.const("AUTUMN", int_t, 0, "COLORMAP_AUTUMN"),
+    gen.const("BONE", int_t, 1, "COLORMAP_BONE"),
+    gen.const("JET", int_t, 2, "COLORMAP_JET"),
+    gen.const("WINTER", int_t, 3, "COLORMAP_WINTER"),
+    gen.const("RAINBOW", int_t, 4, "COLORMAP_RAINBOW"),
+    gen.const("OCEAN", int_t, 5, "COLORMAP_OCEAN"),
+    gen.const("SUMMER", int_t, 6, "COLORMAP_SUMMER"),
+    gen.const("SPRING", int_t, 7, "COLORMAP_SPRING"),
+    gen.const("COOL", int_t, 8, "COLORMAP_COOL"),
+    gen.const("HSV", int_t, 9, "COLORMAP_HSV"),
+    gen.const("PINK", int_t, 10, "COLORMAP_PINK"),
+    gen.const("HOT", int_t, 11, "COLORMAP_HOT"),
+], "colormap")
+
 gen.add('do_dynamic_scaling', bool_t, 0, 'Do dynamic scaling about pixel values or not', False)
+gen.add('colormap', int_t, -1, "colormap", -1, -1, 11, edit_method=edit_method_colormap);
 
 exit(gen.generate(PACKAGE, 'image_view', 'ImageView'))

--- a/image_view/src/nodes/image_view.cpp
+++ b/image_view/src/nodes/image_view.cpp
@@ -50,11 +50,13 @@ boost::format g_filename_format;
 boost::mutex g_image_mutex;
 std::string g_window_name;
 bool g_do_dynamic_scaling;
+int g_colormap;
 
 void reconfigureCb(image_view::ImageViewConfig &config, uint32_t level)
 {
   boost::mutex::scoped_lock lock(g_image_mutex);
   g_do_dynamic_scaling = config.do_dynamic_scaling;
+  g_colormap = config.colormap;
 }
 
 void imageCb(const sensor_msgs::ImageConstPtr& msg)
@@ -65,6 +67,7 @@ void imageCb(const sensor_msgs::ImageConstPtr& msg)
   try {
     cv_bridge::CvtColorForDisplayOptions options;
     options.do_dynamic_scaling = g_do_dynamic_scaling;
+    options.colormap = g_colormap;
     g_last_image = cv_bridge::cvtColorForDisplay(cv_bridge::toCvShare(msg), "",
                                                  options)->image;
   } catch (cv_bridge::Exception& e) {

